consumer/dvo_rules_consumer_test.go

```diff
@@ -304,15 +304,4 @@ func TestProcessEmptyDVOMessage(t *testing.T) {
 	// message is empty -> nothing should be written into storage
 	err := c.HandleMessage(&message)
 	assert.EqualError(t, err, "unexpected end of JSON input")
-
```

Comment:
don't we have an alternative for this but on the DVO tables so that we don't lose this check?

Reply:
I spoke with Dan and he told me that we sadly dont

Reply:
not yet, I plan to add it with my next changes